### PR TITLE
Added PS steps to create-cloud-based-archive.md

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2476

### CODE REVIEW

**Comment by dariomws** [comment]
@nam31, I want to add @get-itips as a reviewer here. Is that you for you? Thanks

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-deployment/create-cloud-based-archive.md [hunk 1]
```diff
@@ -82,7 +78,31 @@ Use one of the following procedures to enable a cloud-based archive mailbox for
...
+
+The following example creates a new primary **online mailbox** and Active Directory user account for Kim Akers with a cloud-based archive mailbox:
+
+```PowerShell
+New-RemoteMailbox -Name "Kim Akers" -UserPrincipalName kima@contoso.com -Password (ConvertTo-SecureString -String 'Pa$$word1' -AsPlainText -Force) -Archive -FirstName Kim -LastName Akers
+```
+
+For detailed syntax and parameter information, see [New-RemoteMailbox](https://docs.microsoft.com/powershell/module/exchange/new-remotemailbox).
+
+After you create the primary and cloud-based archive mailboxes, wait up to 30 minutes for directory synchronization to create a corresponding user account in Microsoft 365 or Office 365. Then assign the product license as we described in the steps 8 to 10 of [Use the EAC to create a cloud-based archive mailbox for a new user](#use-the-eac-to-create-a-cloud-based-archive-mailbox-for-a-new-user). To use PowerShell to assign the license, see [Assign Microsoft 365 licenses to user accounts with PowerShell](https://docs.microsoft.com/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/create-cloud-based-archive.md:99]:
```suggestion
After you create the primary and cloud-based archive mailboxes, wait up to 30 minutes for directory synchronization to create a corresponding user account in Microsoft 365 or Office 365. Then assign the product license as we described in steps 8 to 10 of [Use the EAC to create a cloud-based archive mailbox for a new user](#use-the-eac-to-create-a-cloud-based-archive-mailbox-for-a-new-user). To use PowerShell to assign the license, see [Assign Microsoft 365 licenses to user accounts with PowerShell](https://docs.microsoft.com/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell).
```

File: Exchange/ExchangeHybrid/hybrid-deployment/create-cloud-based-archive.md [hunk 2]
```diff
@@ -121,17 +141,42 @@ Use one of the following procedures to enable a cloud-based archive mailbox for
     > [!TIP]
     > In the admin center, go to **Health** \> **Directory sync status** to see the last time that directory synchronization occurred.
 
+### Use the Exchange Management Shell to create a cloud-based archive mailbox for an existing user
+
+Before you create the cloud-based archive mailbox, you need to assign the product license as we described in the steps 1 to 4 of [Use the EAC to create a cloud-based archive mailbox for an existing user](#use-the-EAC-to-create-a-cloud-based-archive-mailbox-for-an-existing-user). To use PowerShell to assign the license, see [Assign Microsoft 365 licenses to user accounts with PowerShell](https://docs.microsoft.com/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/create-cloud-based-archive.md:146]:
```suggestion
Before you create the cloud-based archive mailbox, you need to assign the product license as we described in Steps 1 to 4 of [Use the EAC to create a cloud-based archive mailbox for an existing user](#use-the-EAC-to-create-a-cloud-based-archive-mailbox-for-an-existing-user). To use PowerShell to assign the license, see [Assign Microsoft 365 licenses to user accounts with PowerShell](https://docs.microsoft.com/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Review by get-itips — APPROVED** [review]

**Comment by nam31** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**